scripts/ci/build_environment.sh

```diff
@@ -0,0 +1,6 @@
+conda install -c conda-forge mamba
+conda env create -f environment.yml
+conda activate nams || source activate nams
```

Comment:
```suggestion
conda activate jax-unirep || source activate jax-unirep
```

Is it weird that the build is passing even though the env name was wrong here?

Reply:
That is a great catch! Yet again, I'm caught copy/pasting things lol.

Yes, it's weird, but let's see what happens after committing the changes.